package.json

```
@@ -16,12 +16,14 @@
   },
   "author": "Iain Collins <me@iaincollins.com>",
   "contributors": [
-    "Balázs Orbán <info@balazsorban.com>"
+    "Balázs Orbán <info@balazsorban.com>",
+    "Nico Domino <yo@ndo.dev>"
   ],
   "license": "ISC",
   "dependencies": {
-    "next": "^11.0.0",
-    "next-auth": "latest",
+    "next": "^12.0.1",
+    "next-auth": "^4.0.0-beta.7",
```

```suggestion
    "next-auth": "4.0.0-beta.7",
```

Use either exact version, or `beta` for now.

---

Good catch!